resources/js/features/user/settings/components/+root/SettingsRoot.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react';
+
+import { ChangeEmailAddressSectionCard } from '../ChangeEmailAddressSectionCard';
+import { ChangePasswordSectionCard } from '../ChangePasswordSectionCard';
+import { DeleteAccountSectionCard } from '../DeleteAccountSectionCard';
+import { KeysSectionCard } from '../KeysSectionCard';
+import { NotificationsSectionCard } from '../NotificationsSectionCard';
+import { PreferencesSectionCard } from '../PreferencesSectionCard';
+import { ProfileSectionCard } from '../ProfileSectionCard';
+import { ResetGameProgressSectionCard } from '../ResetGameProgressSectionCard';
+
+export const SettingsRoot: FC = () => {
+  return (
+    <div className="flex flex-col">
+      <h1>Settings</h1>
+
+      <div className="flex flex-col gap-4">
+        <ProfileSectionCard />
+        <NotificationsSectionCard />
+        <PreferencesSectionCard />
```

Comment:
If changes are made to Notifications, and updated, then changes are made to Preferences and updated, the changes to Notifications are overwritten (only the changes to Preferences will be in the database).

This behavior does not occur on the controlpanel.php page.

Reply:
Good catch, I've fixed this in latest.